src/sidebar/components/test/annotation-test.js

```diff
@@ -1068,22 +1068,57 @@ describe('annotation', function() {
     });
 
     describe('tag display', function () {
-      it('displays links to tags on the stream', function () {
+      beforeEach('make serviceUrl() return a URL for the tag', function() {
         fakeServiceUrl
           .withArgs('search.tag', {tag: 'atag'})
           .returns('https://test.hypothes.is/stream?q=tag:atag');
+      });
 
-        var directive = createDirective(Object.assign(fixtures.defaultAnnotation(), {
+      /*
+       * Return an annotation directive with a single tag.
```

Comment:
Technically it is a "component" now rather than a directive. I'm guessing there are quite a lot of references in this file which haven't been updated.

Reply:
:(